### DISCUSSION TITLE
Contributions: основные вопросы

### DISCUSSION THREAD

**Discussion by hoshion** [HowProgrammingWorks/LiveQA#41] [Q&A] [answered]
Хотелось бы узнать по подробней по поводу контрибьюшенов в Open Source проекты и зачитывание этого как оценку за предмет, а именно:

- в какие проекты можно контрибьютить?
- что можно контрибьютить?
- сколько можно получить баллов/есть ли градация по баллам, в зависимости от самого контрибьюта?

**Comment by tshemsedinov** [comment, marked as answer]
Я это уже объяснял
- В любые проекты
- Все, что хотите
- Когда я увижу, что вы сделали, только тогда смогу оценить
